apps/music/css/view.css

```diff
@@ -15,17 +15,24 @@ body {
   -moz-user-select: none;
 }
 
-gaia-fast-list em {
+/**
+ * 1. List-item height
+ */
+
+a[number]:before {
```

Comment:
We should probably use `a[data-number]` since we shouldn't define new attributes on existing elements (non-WC's)